crates/blockchain-tree/src/blockchain_tree.rs

```diff
@@ -336,7 +336,7 @@ impl<DB: Database, C: Consensus, EF: ExecutorFactory> BlockchainTree<DB, C, EF>
         parent: BlockNumHash,
     ) -> Result<BlockStatus, InsertBlockError> {
         let block_num_hash = block.num_hash();
-        debug!(target: "blockchain_tree", ?parent, "Appending block to canonical chain");
+        debug!(target: "blockchain_tree", head=?block_num_hash.hash, ?parent, "Appending block to canonical chain");
```

Comment:
nit: let's keep it consistent
```suggestion
        debug!(target: "blockchain_tree", head = ?block_num_hash.hash, ?parent, "Appending block to canonical chain");
```